source/documentation/other-topics/route53-zone.html.md.erb

```diff
@@ -34,28 +35,69 @@ provider "aws" {
 ```
 Note: If you already have that file defined in your environment, do not recreate it.
 
+### variable.tf
+
+Fill in the variable  _namespace_  field below with your existing kubernetes namespace.
+
+```
+variable "namespace" {
+  default = "YOUR KUBERNETES NAMESPACE GOES HERE"
+}
+
+variable "business-unit" {
+  default = "example-bu"
+}
+
+variable "team_name" {
+  default = "example-team-name"
+}
+
+variable "application" {
+  default = "example-app"
+}
+
+variable "environment-name" {
+  default = "dev"
+}
+
+variable "is-production" {
+  default = "false"
+}
+
+variable "infrastructure-support" {
+  default     = "example@example.com"
+  }
+
+variable "owner" {
+  default     = "example-owner"
+  }
+
+```
+
```

Comment:
@pwyborn , is it worth adding the line `Note: If you already have that file defined in your environment, do not recreate it.` here similar to `main.tf`?

Reply:
Yes - good idea Poornima @poornima-krishnasamy - now changed.
I also added a line about what the variable.tf is for?